lib/workers/branch/commit.ts

```diff
@@ -31,9 +31,20 @@ export function commitFilesToBranch(
   }
   const fileLength = [...new Set(updatedFiles.map((file) => file.name))].length;
   logger.debug(`${fileLength} file(s) to commit`);
-  // istanbul ignore if
   if (getAdminConfig().dryRun) {
-    logger.info('DRY-RUN: Would commit files to branch ' + config.branchName);
+    logger.info(
+      {
+        commitMessage: config.commitMessage,
+
+        // Piping f.contents to
+        // git diff -- ${f.name} -
+        // would be much better, but this seems to be impossible with simple-git
```

Comment:
```suggestion
```

`git diff` won't work, as not all updates are already written to disk at that time

Reply:
Maybe for a future PR, but we could potentially pass dryRun to the git layer and let it progress all the way until the `git push`.

Reply:
Sounds good to me

Reply:
Doesn't `f.contents` have the contents that will be committed? We could test if the file exists, and print the diff if it does and `f.contents if it doesn't

Reply:
I think passing to the git layer and debugging there is much better then inventing our own diff (which could include binary files)